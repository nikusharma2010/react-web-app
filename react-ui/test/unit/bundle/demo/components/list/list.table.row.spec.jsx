import React from "react";
import { create } from "react-test-renderer";
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { MemoryRouter } from 'react-router-dom';
import ListTableRow from "../../../../../../src/bundle/demo/components/list/list.table.row";
Enzyme.configure({ adapter: new Adapter() });

describe("List Row Component", () => {
    test("it matches the snapshot", () => {
        const props = {
            user: {
                "id": "7",
                "firstName": "Niku",
                "lastName": "Sharma",
                "email": "7@thesimpsons.com",
                "mobile": "007"
            }
        };
        const component = create(<MemoryRouter><ListTableRow {...props} /></MemoryRouter>);
        expect(component.toJSON()).toMatchSnapshot();
    });
});
