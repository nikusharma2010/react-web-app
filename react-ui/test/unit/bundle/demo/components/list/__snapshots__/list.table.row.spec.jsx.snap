// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Row Component it matches the snapshot 1`] = `
<div
  className="styles__Row-sc-18xwbcb-3 cYIdhM"
>
  <div
    className="styles__Col-sc-18xwbcb-2 eBtAHL"
  >
    <div
      className="styles__NameLabel-sc-18xwbcb-5 dErOaa"
    >
      N
      S
    </div>
  </div>
  <div
    className="styles__Col-sc-18xwbcb-2 jTVgQF"
  >
    <div>
      <b>
        Niku Sharma
      </b>
    </div>
    <div>
      007
    </div>
    <div>
      7@thesimpsons.com
    </div>
  </div>
  <div
    className="styles__Col-sc-18xwbcb-2 gAVkdx"
  >
    <a
      href="/view/7"
      onClick={[Function]}
    >
      <div
        className="styles__Button-sc-18xwbcb-6 styles__ButtonPrimary-sc-18xwbcb-7 fKWqEi"
      >
        View
      </div>
       
    </a>
  </div>
</div>
`;
